Base/QTGUI/Testing/Cxx/qSlicerMouseModeToolBarTest1.cxx

```diff
@@ -60,67 +75,11 @@ int qSlicerMouseModeToolBarTest1(int argc, char * argv[] )
 
   QString activeActionText;
   activeActionText = activePlaceActionText(mouseToolBar);
-  std::cout << "Active action text = " << qPrintable(activeActionText) << std::endl;
+  std::cout << "Active place action text = " << qPrintable(activeActionText) << std::endl;
 
-  // get the selection and interaction nodes that the mouse mode tool bar
+  // get interaction nodes that the mouse mode tool bar
   // listens to
   vtkMRMLNode *mrmlNode;
-  vtkMRMLSelectionNode *selectionNode = nullptr;
```

Comment:
The mouse mode toolbar test has been updated to check for the default button state (toggle the Markups button), and default cursor (translate/rotate). I can add the a check for the WindowLevel mode. Testing the placement mode will require adding markup nodes to the scene, which I think would be better added as a test of the new Markups Toolbar, does this seem correct?

Reply:
It would be better to test this toolbar (including placement mode) independently from the markups toolbar. Just adding nodes to the scene and modifying the selection node should update the toolbar's state correctly.